udata/core/activity/api.py

```diff
@@ -79,6 +85,9 @@ def get(self):
         if args["user"]:
             qs = qs(actor=args["user"])
 
+        if args["related_to"]:
+            qs = qs(related_to=args["related_to"])
```

Comment:
We may want to test for a valid ObjectId (ex using `ObjectId.is_valid`) to prevent an internal server error.